libs/durability/near_full_data_storage.py

```diff
@@ -184,10 +184,11 @@ def delete_workload(workload_info_list: list, s3userinfo: dict, delete_percent:
         num_buckets_delete = int(delete_percent * len(workload_info_list) / 100)
         delete_list = []
         LOGGER.info("Delete %s random buckets.", num_buckets_delete)
-
+        LOGGER.debug("workload info: %s", workload_info_list)
+        LOGGER.debug("Number of available buckets : %s", len(workload_info_list))
```

Comment:
This debug statement can be added after bucket deletion.

Reply:
made changes